### PR TITLE
news: add entries for #3555

### PR DESCRIPTION
**Pull request by MrAnno**

### CODE REVIEW

**Review by Kokan — APPROVED** [review]

File: news/create-newsfile.py [hunk 1]
```diff
@@ -62,10 +62,12 @@ def create_block(block_name, files):
     block = '## {}\n\n'.format(block_name)
     for f in files:
         entry = ''
-        try:
-            pr_id = re.findall(r'\d+.md$', f.name)[0][:-3]
-        except IndexError:
+        match_pr_id = re.search(r'(\d+)(-\d+)?.md$', f.name)
```
Comment (alltilla) [inline news/create-newsfile.py:65]: Please update the README, to document this possibility, thanks :)

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: news/create-newsfile.py (hunk 1 above)
Comment (MrAnno) [reply news/create-newsfile.py:65]: Done, thank you.

**Review by alltilla — APPROVED** [review]

File: news/create-newsfile.py (hunk 1 above)
Comment (alltilla) [reply news/create-newsfile.py:65]: Thanks!

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by Kokan — APPROVED** [review]

**Pull request merged**